web_src/js/components/RepoActionView.vue

```diff
@@ -43,11 +43,18 @@
 
       <div class="action-view-right">
         <div class="job-info-header">
-          <div class="job-info-header-title">
-            {{ currentJob.title }}
+          <div class="job-info-header-name">
+            <div class="job-info-header-title">
+              {{ currentJob.title }}
+            </div>
+            <div class="job-info-header-detail">
+              {{ currentJob.detail }}
+            </div>
           </div>
-          <div class="job-info-header-detail">
-            {{ currentJob.detail }}
+          <div class="job-info-header-props">
+            <a class="job-brief-link" :href="run.link+'/jobs/'+jobIndex+'/logs'" target="_blank">
```

Comment:
```suggestion
            <a class="job-brief-link muted" :href="run.link+'/jobs/'+jobIndex+'/logs'" target="_blank">
```

Reply:
Thanks, but I see there's a menu in the rebased version, so I added it as a menu item

Reply:
I re-added `target="_blank"` because it prevents you from losing your tab if the server returns error

Reply:
I guess you want to instead set the [`download`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download) attribute then, likely in boolean form. `target=_blank` is not right for downloads, this one is.

Reply:
Done that in https://github.com/go-gitea/gitea/pull/24451/commits/5e29e0592b51f39742e1c757e50e5a5fd6da3860, now there's no more flash of a tab opening and closing.